src/libraries/Microsoft.Extensions.Hosting.WindowsServices/tests/WindowsServiceLifetimeTests.cs

```diff
@@ -194,6 +198,8 @@ lifetime stopped
                 """, logText);
         }
 
+        private static AutoResetEvent are = new AutoResetEvent(false);
```

Comment:
nit: use `s_are`

Reply:
Use 2 different events?

Reply:
> nit: use s_are

Done

> Use 2 different events?

Updated the windows service `OnStop()` to wait only for the `ServiceSequenceIsCorrect` test